.github/CODEOWNERS

```diff
@@ -3,8 +3,8 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @aloftus23 @cduhn17 @dav3r @DJensen94 @elr64 @felddy @jsf9k @mcdonnnj @schmelz-ctr @stewartl97
+* @aloftus23 @cduhn17 @dav3r @DJensen94 @elr64 @felddy @jasonodoom @jsf9k @mcdonnnj @schmelz-ctr @stewartl97
```

Comment:
@elr64 and @schmelz-ctr no longer exist as GitHub users.  I suspect they should be removed.

Reply:
@cduhn17 or @aloftus23 - can you please confirm that elr64 and schmelz-ctr should be removed as code owners here or point us in the direction of someone who can?

Reply:
@cduhn17 and @aloftus23 - Bumping this for visibility.

Reply:
@dav3r , 
Yes you can remove @schmelz21 and @elr64 as code owners as they have moved on from P&E

Reply:
Removed in commit a5acb81.